charts/super-agent-deployment/values.yaml

```diff
@@ -116,6 +128,12 @@ config:
       #    endpoint: https://opamp.service.newrelic.com/v1/opamp
       #    headers:
       #      api-key: LICENSE_KEY
+      #    auth_config:
+      #      token_url: "http://fake.com/oauth2/v1/token"
+      #      client_id: "fake"
+      #      provider: "local"
```

Comment:
provider? which values are supported?

Reply:
I added all the exposed fields from -> https://github.com/newrelic/newrelic-super-agent/blob/main/super-agent/src/opamp/auth/config.rs#L11, bud since it has the `deserialize_default_provider` I can just ignore the provider 🤔 (however it's still exposed if set)

Reply:
🤔 ah ok, so right now there is only "local" and we are going to have something else in the future...

your call I am fine having or not having it for now

Reply:
I just removed `provider`.